### DISCUSSION TITLE
Duplicate new item templates

### DISCUSSION THREAD

**Discussion by moon6969** [AvaloniaUI/Avalonia#12835] [Q&A]
I've just installed Avalonia 11 templates in VS2022 and are a little confused by the duplicate templates (Resource Dictionary, Styles, etc.). One set has a generic b&w icon and the other the purple Avalonia icon.

The only obvious difference is the ~~former~~ latter doesn't include the subfolder in the namespace...
`namespace Avalonia.MusicStore;`
vs
`namespace Avalonia.MusicStore.Views`

Is this just a hangover from 0.10?

![image](https://github.com/AvaloniaUI/Avalonia/assets/30491315/ba00f7c3-87ab-4fca-8051-21afffb9f55a)

**Comment by moon6969** [comment]
It looks like the b&w icon templates are installed with `Avalonia for Visual Studio 2022` extension which also has a single project template "Avalonia C# Project" which is wizard based.

`dotnet new install Avalonia.Templates` adds the purple icon item templates as well as 3 non-wizard project templates.

Still not sure how to choose which template to use.

**Comment by maxkatz6** [comment]
cc @Takoooooo
